client/src/pages/SubscriptionPage.jsx

```diff
@@ -0,0 +1,102 @@
+import { useState, useEffect } from "react";
+import Subscription from "../components/Subscription";
+
+const textLabel = [
+  {
+    type: "text",
+    id: "firstname",
+    for: "firstname",
+    text: "Prénom",
+  },
+  {
+    type: "text",
+    id: "lastname",
+    for: "lastname",
+    text: "Nom",
+  },
+  {
+    type: "email",
+    id: "email",
+    for: "email",
+    text: "Adresse mail",
+  },
+  {
+    type: "password",
+    id: "password",
+    for: "password",
+    text: "Mot de passe",
+  },
+];
+
+const siret = {
+  type: "text",
+  id: "siret",
+  for: "siret",
+  text: "N° de SIRET",
+};
+
+const emptyFields = {
+  firstname: "",
+  lastname: "",
+  email: "",
+  password: "",
+  siret: "",
+};
+
+function SubscriptionPage() {
+  const [fields, setFields] = useState(textLabel);
+  const [formValues, setFormValues] = useState(emptyFields);
+
+  const handleClickCustomer = () => {
```

Comment:
Code is always duplicated between handleClickCustomer and handleClickProfessional